org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -149,6 +151,8 @@ class Translator {
         validateArgumentType(deviceInfo, "deviceInfo", "object");
         
         this.controlId = deviceInfo.deviceInfo.opent2t.controlId;
+        this.uuid = deviceInfo.deviceInfo.opent2t.uuid;
+        console.log("Translator:"+JSON.stringify(deviceInfo.deviceInfo));

```

Comment:
Fixed.

---
In reply to: [114233414](https://github.com/openT2T/translators/pull/237#discussion_r114233414) [](ancestors = 114233414)